cmd/storj-admin/main.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2020 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+
+	"storj.io/common/fpath"
+	"storj.io/private/cfgstruct"
+	"storj.io/private/process"
+	_ "storj.io/storj/private/version" // This attaches version information during release builds.
+)
+
+var (
+	rootCmd = &cobra.Command{
+		Use:   "storj-admin",
+		Short: "A tool for managing operations against a satellite",
+	}
+	setupCmd = &cobra.Command{
+		Use:         "setup",
+		Short:       "Create config files",
+		RunE:        cmdSetup,
+		Annotations: map[string]string{"type": "setup"},
+	}
+	runCmd = &cobra.Command{
+		Use:   "run",
+		Short: "Run the storj-admin",
+		//RunE:  cmdRun,
```

Comment:
Does this mean we don't need this `runCmd`?

Reply:
It comes in with another PR. Its part of #3887 already